lib/kubernetes-deploy/kubernetes_resource/pod_disruption_budget.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+module KubernetesDeploy
+  class PodDisruptionBudget < KubernetesResource
+    TIMEOUT = 30.seconds
```

Comment:
I think it takes way less time to apply PodDisruptionBudget? Does it have to be 30 seconds? I'm asking because it sounds inefficient to sleep for 30 seconds if `PodDisruptionBudget` failed to apply.